use/2016-01-26-SNP-selection.Rmd

```diff
@@ -100,12 +100,12 @@ discovery rate (q-value) to provide a list of outliers.
 genotype <- sel[, 3:ncol(sel)]
 dim(genotype)
 K <- 25
-x <- pcadapt(data = genotype, K = K)
+x <- pcadapt(genotype, K = K, transpose = TRUE)
```

Comment:
I'm wondering whether it's obvious that the `transpose = TRUE` parameter is here because unlike others _pcadapt_ now expects the variants in rows. Isn't this sufficiently peculiar to deserve a brief mention either in the text, or perhaps just as a comment in the R code snippet?